### PR TITLE
Let sequencer type be configurable

### PR DESCRIPTION
**Pull request by baldersheim**
@hmusum or @vekterli PR

### CODE REVIEW

**Review by hmusum — APPROVED** [review]
LGTM

**Review by vekterli — APPROVED** [review]
:+1:

**Comment by hmusum** [comment]
Some unit tests failed

File: flags/src/main/java/com/yahoo/vespa/flags/Flags.java [hunk 1]
```diff
@@ -151,6 +151,16 @@
...
             "Takes effect at redeployment",
             ZONE_ID, APPLICATION_ID);
+    public static final UnboundStringFlag RESPONSE_SEQUENCER_TYPE = defineStringFlag(
+            "response-sequencer-type", "ADAPTIVE",
+            "Selects type of sequenced executor used for mbus responses, valid values are LATENCY, ADAPTIVE, THROUGHPUT",
+            "Takes effect at redeployment",
+            ZONE_ID, APPLICATION_ID);
+    public static final UnboundIntFlag RESPONSE_NUM_THREADS = defineIntFlag(
+            "response-num-threads", 2,
+            "Selects type of sequenced executor used for mbus responses, valid values are LATENCY, ADAPTIVE, THROUGHPUT",
```
Comment (hmusum) [inline flags/src/main/java/com/yahoo/vespa/flags/Flags.java:161]: Duplicated description of added flag above

**Review by hmusum — APPROVED** [review]
One minor comment, otherwise LGTM

File: flags/src/main/java/com/yahoo/vespa/flags/Flags.java (hunk 1 above)
Comment (baldersheim) [reply flags/src/main/java/com/yahoo/vespa/flags/Flags.java]: Thanks, Fixed

**Review by hmusum — APPROVED** [review]

**Pull request merged**